packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md

```diff
@@ -0,0 +1,28 @@
+Update the `showUserInfo` function of `ContentView`:
```

Comment:
Perhaps:

Display the user info by updating the...

Reply:
updated